_posts/2025-08-27-offer-multiple-ways-to-contact-you.md

```diff
@@ -0,0 +1,125 @@
+
+Some callers even get hung up on if they take too long to respond. Wouter Honselaar, who stutters, said in the interview [I am stronger than stuttering](https://www.a11y-collective.com/blog/interview-with-wouter-i-am-stronger-than-stuttering/):
+
+> “Once, someone ended the call because they thought the line had gone dead.”
+
+Also, phone calls often only work during office hours. If you work during those hours and can’t make personal calls, it’s not an option.
+
+I asked my deaf followers on LinkedIn:
+
+> “What do you do when the only way to contact a website is by phone, and you *really* need to get in touch?”
+
+[Marie Van Driessche](https://www.linkedin.com/in/marie-van-driessche/) replied:
+
+> “I use a [video relay service](https://en.wikipedia.org/wiki/Video_relay_service) via the Tolkcontact app. So I get a translator to call them for me. It works, but I have to arrange it all myself. It’d be nice if email or WhatsApp were options.”
+
+[Meryl Evans](https://www.linkedin.com/in/meryl/) added:
+
+> “Offer text-based options like email, SMS, live chat with a real person, or a video relay. Let people choose what contact details they share. Don’t make phone numbers required.”
+
+### Email
+
+The nice thing about email is that you can send one whenever you want.
+
+But not everyone finds it easy to write an email, especially if they’re not confident in your langauge, have dyslexia or low literacy. Some people may need help just to write a message.
+
+### Contact Form
+
+Contact forms are handy, you ask exactly what you need, and users can fill it out in an organized way.
+
+But forms can be a real barrier for people who aren’t great with tech. Especially if the form is long or complicated. Always test your forms with real users, including people with disabilities.
+
+If someone is blind, visually impaired, or just not tech-savvy, a poorly set up and coded form can be a nightmare.
+
+So, make sure it’s easy to use and it’s accessible.
+
+### Chat box
+
+Quick question? Don’t want to call? A chat box sounds perfect.
+
+Until you get stuck with a robot that can’t help and just tells you to “check the FAQs” or to call someone. Instead of helping, you're fobbing people off.
+
+Deaf and hard-of-hearing people say chat can be a great alternative to phone calls, as long as it’s a real person.
+
+If you offer chat, let people (maybe after a quick filter) talk to a real human. And show when someone’s available.
+
+### In person contact
+
+Sometimes, like when renewing a passport, you have to show up in person to confirm your identity.
+
+But what if your health makes that impossible? Then what?
+
+Therefore, make sure you offer an alternative. As a municipality you can offer a home visit to confirm the identity of a resident.
```

Comment:
"Make sure you offer an alternative. As a municipality you can offer a home visit to confirm the identity of a resident."